arangod/Aql/ClusterNodes.cpp

```diff
@@ -92,7 +93,46 @@ RemoteNode::RemoteNode(ExecutionPlan* plan, arangodb::velocypack::Slice const& b
 /// @brief creates corresponding ExecutionBlock
 std::unique_ptr<ExecutionBlock> RemoteNode::createBlock(
     ExecutionEngine& engine, std::unordered_map<ExecutionNode*, ExecutionBlock*> const&) const {
-  return std::make_unique<RemoteBlock>(&engine, this, server(), ownName(), queryId());
+  // TODO I'm assuming here that input and output registers are the same for
```

Comment:
Yes they are.
The remote can neither discard nor add something.
Please remove this TODO